assets/constants.ts

```diff
@@ -5353,6 +5353,9 @@ export = {
   // Used to disable account data export until it is fully implemented.
   "ENABLE_ACCOUNT_EXPORT": false,
 
+  // Link to open when the Oppia avatar is clicked on any page.
+  "OPPIA_AVATAR_IMAGE_LINK": null,
```

Comment:
Can you change this to OPPIA_AVATAR_LINK_URL throughout? (Other properties in this file which reference URLs are named as "URL".)

(Only do this if you need to make a commit to handle merge conflicts anyway.)

Reply:
Done